ansible/roles/hoover/tasks/main.yml

```diff
@@ -0,0 +1,103 @@
+---
+- apt_repository:
+    repo: "{{item}}"
+    filename: multiverse
+  with_items:
+    - 'deb http://ports.ubuntu.com/ubuntu-ports {{ansible_distribution_release}} multiverse'
+    - 'deb http://ports.ubuntu.com/ubuntu-ports {{ansible_distribution_release}}-updates multiverse'
+  when: ansible_distribution == 'Ubuntu'
+
+- name: Install required system packages
+  apt: name={{ item }} state=latest
+  with_items:
+    - python-psycopg2
+    - libmagic-dev
+    - python3-dev
+    - libxml2-dev
+    - libxslt1-dev
+    - p7zip-full
+    - p7zip-rar
+    - pst-utils
+    - cpanminus # for cpanm ansible module
+
+- name: Install msgconvert from CPAN
+  cpanm:
+      name: Email::Outlook::Message
+
+- name: Run hoover initial setup
+  shell: "python3 <(curl -sL https://github.com/hoover/setup/raw/automate-hoover-script/install.py)"
+  environment:
+    HOOVER_HOME: "/opt/hoover"
+    HOOVER_BOOTSTRAP_NO_DB: "True"
+    HOOVER_ALLOWED_HOSTS: "hoover.{{ liquid_domain }}"
+    HOOVER_SETUP_BRANCH: "automate-hoover-script"
+    HOOVER_ELASTICSEARCH_URL: "http://localhost:14352"
+    HOOVER_TIKA_URL: "http://localhost:15423"
+  args:
+    executable: /bin/bash
+    chdir: /opt
+    creates: /opt/hoover
```

Comment:
If the task fails, can we please remove the `/opt/hoover` folder, so the next ansible run won't skip this step? I think [block error handling](http://docs.ansible.com/ansible/playbooks_blocks.html#error-handling) might do the trick.